frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx

```diff
@@ -181,24 +186,38 @@ export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
         );
         if (foundField?.legalValues && foundField.legalValues.length > 0) {
             const options = foundField.legalValues.map(({ value }) => value);
+
             return (
                 <Autocomplete
-                    disablePortal
-                    limitTags={3}
-                    id='context-legal-values'
-                    freeSolo
-                    filterSelectedOptions
-                    size='small'
-                    value={resolveAutocompleteValue()}
-                    onChange={changeContextValue}
-                    options={options}
-                    multiple={true}
-                    sx={{ width: 370, maxWidth: '100%' }}
-                    renderInput={(params: any) => (
-                        <TextField {...params} label='Value' />
-                    )}
-                    disableCloseOnSelect={false}
-                />
+                disablePortal
+                limitTags={3}
+                id='context-legal-values'
+                multiple={true}
+                options={options}
+                disableCloseOnSelect
+                size='small'
+                value={resolveAutocompleteValue()}
+                onChange={changeContextValue}
+                getOptionLabel={(option) => option}
+                renderOption={(props, option, { selected }) => {
+                    const { key, ...optionProps } = props;
+                    return (
+                        <li key={key} {...optionProps}>
+                            <Checkbox
+                                icon={<CheckBoxOutlineBlank fontSize='small' />}
+                                checkedIcon={<CheckBoxIcon fontSize='small' />}
+                                style={{ marginRight: 0.4 }}
```

Comment:
```suggestion
                                    sx={theme => ({ marginRight: theme.spacing(0.5) })}
```

Reply:
Hi @Tymek , I have fixed the linter issues and addressed your feedback. Let me know if there's anything else needed!